controller/test/blower_fsm/blower_fsm_test.cpp

```diff
@@ -62,17 +66,35 @@ void testSequence(
                    /*expected_pressure_setpoint*/ std::optional<Pressure>,
                    /*expected_flow_direction*/ FlowDirection>> &seq) {
   BlowerFsm fsm;
+  if (seq.empty()) {
+    return;
+  }
+
+  // Reset time to test's start time.
+  Hal.delay(microseconds(-Hal.now().microsSinceStartup()));
```

Comment:
Uh a negative delay looks super awkward. Can we do without it? (OK to leave a TODO explaining how)